### DISCUSSION TITLE
DAI 2024-2025 - Practical work 1 - Mathieu Emery and Tristan Baud

### DISCUSSION THREAD

**Discussion by NATSIIRT** [heig-vd-dai-course/heig-vd-dai-course#347] [Show and tell]
The project will concern the creation of a CLI to encrypt and decrypt files using different algorithms.

Members :

- Mathieu Emery ([@mathieuemery](https://github.com/mathieuemery))
- Tristan Baud  ([@NATSIIRT](https://github.com/NATSIIRT))

Repository : [DAI-LAB-02](https://github.com/HEIG-VD-S3-PW/DAI-LAB-02)
Final commit hash : [c3704db5cf488cf784a8cf1664c376f5227b2374](https://github.com/HEIG-VD-S3-PW/DAI-LAB-02/commit/c3704db5cf488cf784a8cf1664c376f5227b2374)

**Comment by ludelafo** [comment]
All good for us! Thank you!
